src/extension.ts

```diff
@@ -248,6 +249,11 @@ export function activate(context: vscode.ExtensionContext): void {
         }
 
         await stopStreamingLogs(node.root.client);
+        if (node.contextValue === LogStreamTreeItem.contextValue) {
+            // don't wait for this refresh
+            // tslint:disable-next-line:no-floating-promises
+            node.refresh();
```

Comment:
Why don't wait?

Reply:
I figured we wouldn't want to hang up in case the refresh took a while.  Figured there was nothing to gain from awaiting it.